roles/cfg_openwrt/templates/common/config/dsa.network.inc

```diff
@@ -6,6 +6,9 @@ config device
   {% set portmapping = [] %}
```

Comment:
These port_untag changes should probably live in a separate commit too.

Reply:
It is? https://github.com/freifunk-berlin/bbb-configs/pull/195/commits/e0b565ee227d557160f4e79e7120c30c87a9facc

Reply:
Oupsi